### PR TITLE
Feat/remd aniso

### PR DESCRIPTION
**Pull request by ceriottm**
Mixed bugfixes and improvements from constant-pressure simulations

Started as a quick-fix of parrinello-rahman, ended up adding a lot of features and fixing a log of nasty bugs

### CODE REVIEW

**Comment by mahrossi** [comment]
Hi there @venkatkapil24 and @ceriottm I will review other PRs that are open but since these are many bug fixes, could you take care and merge? Pretty important to do before Thursday... ;)

File: ipi/utils/inputvalue.py [hunk 1]
```diff
@@ -862,19 +862,21 @@ def store(self, value, units=""):
         if units != "":
             self.units.store(units)  # User can define in the code the units to be printed
 
-        self.value = value
         if self._dimension != "undefined":
-            self.value *= unit_to_user(self._dimension, units, 1.0)
+            self.value = value * unit_to_user(self._dimension, units, 1.0)
```
Comment (venkatkapil24) [inline ipi/utils/inputvalue.py]: Are these relevant to the branch ?
